src/main/java/com/cloudbees/jenkins/plugins/bitbucket/hooks/PushHookProcessor.java

```diff
@@ -71,13 +70,6 @@ public void process(HookEventType hookEvent, String payload, BitbucketType insta
                 push = BitbucketCloudWebhookPayload.pushEventFromPayload(payload);
             }
             if (push != null) {
-                String owner = push.getRepository().getOwnerName();
```

Comment:
As described in JIRA issue I think these changes will cause to the user that have setup "Discover pull requests" strategies different to "The current pull request revision" to not re-build anymore the PRs target the same merged branch of the event.
If I understand correct the configuration in the JIRA issue the reported would trigger manual reindex to update all PRs. Than this should be archived using a build strategies to ignore some specific index events (maybe already implemented in other plugins)